packages/field/test/FormatMixin.test.js

```diff
@@ -223,9 +223,28 @@ describe('FormatMixin', () => {
       el.errorState = false;
       el.modelValue = 'bar2';
       expect(formatterSpy.callCount).to.equal(2);
-
       expect(el.formattedValue).to.equal('foo: bar2');
     });
+
+    it('will recall the formatter if errorState has changed', async () => {
```

Comment:
Maybe create an it that describes the failing feature, since this is not necessarily a feature (but a temp solution to solve a bug) from the app developer perspective.
Could we add some code to the above test that fails in the prev state? (that doesn't reformat when invlaid input becomes valid)